### ISSUE TITLE
[TODO] Enable HTTP request caching

### ISSUE THREAD

**Issue by webknjaz** [sanitizers/chronographer-github-app#16]
AFAIR this can be wired up on the router level.

**Comment by pradyunsg** [comment]
You might be able to use cache control here.

*might*

**Comment by webknjaz** [comment]
Yeah. I know :) I just didn't do this. There's a native feature in Gidgethub for that.

**Comment by webknjaz** [comment]
I was actually in the wrong repo. This should be https://github.com/sanitizers/octomachinery/issues/17.

**Comment by pradyunsg** [comment]
Hehe. Cool cool. 😬 

My lack of knowledge in this department shows. 🙃

**Comment by webknjaz** [comment]
That's okay ;)
This is only needed as a means to avoid hitting the rate limit. GitHub's docs say that if the resource hasn't changed it'll not be counted towards the used requests counter.

**Comment by webknjaz** [comment]
Pointers: https://github.com/sanitizers/octomachinery/issues/17#issuecomment-515949029
